resources/core/charts/monitoring/charts/alert-rules/README.md

```diff
@@ -2,12 +2,13 @@
 
 ## Overview
 
-In order to provide a starting point for adding alert rules, Kyma includes a helm chart where new rules can be created.
+Kyma uses Prometheus alert rules for monitoring the health of its resources. And this chart is the starting point for configuring alert rules.

```

Comment:
~And~ `This chart is the starting point for configuring alert rules.`

Reply:
changed